spouts/rss/feed.php

```diff
@@ -212,7 +212,7 @@ public function getId() {
      */
     public function getTitle() {
         if ($this->items !== false && $this->valid()) {
-            return @current($this->items)->get_title();
+            return htmlspecialchars_decode(@current($this->items)->get_title());
```

Comment:
Because otherwise, a commit message containing HTML entities would mess up the selfoss stream in the client browser (or be stripped if the tag is not allowed). What we want from commit messages is the raw HTML to be shown in the browser, thus the escaping.

On the contrary, in RSS feeds, we want the HTML to format text in the client browser. Thus, it must be passed as raw HTML the the content loader, filtered by sanitize{Field,Content}() and sent the the client browser.